src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -3496,6 +3556,9 @@ void processContextElement
         //
         ContextElementResponse* notifyCerP = new ContextElementResponse(ceP, apiVersion == "v2");
 
+        // Set metadata flags
+        setCreateAndUpdateFlags(notifyCerP);
```

Comment:
The function name would depend on the name of the flags that the function sets.
Right now these flags are called onUpdate and onChange, so:

  setCreateAndUpdateFlags => setChangeAndUpdateFlags

However, change and update have a very similar meaning.
I believe we should try to find better names of the flags and after that rename the function accordingly.

I believe the variables have the following meaning:
  onChange:  the subscription has been modified
  onUpdate:   the value of the attribute has been modified

If this is true, perhaps **subUpdated** and **valueUpdated** are acceptable names?
The function could be called **setNotificationFlags** (or **setUpdatedFlags**).